dev-tools/reconfigurator-cli/tests/output/cmd-complex-stdout

```diff
@@ -80,51 +80,51 @@ comment: from collection 5e1d6ae0-c5b4-4884-ac41-680cd4f5762d
 zones:
 
   sled 1f34b5cc-6dac-40cc-9ea1-95f32c33e59b: Omicron zones at generation 5
-    0db88baf-911d-47bb-af71-3aebd7c96934 in service     nexus
-    213d7dd3-95a4-45a7-99ca-e4899991dab4 in service     crucible
-    218f36a5-9e51-4b88-ad1a-da4d85c246c4 in service     internal_dns
-    40c47ead-3be8-4724-a8ef-b66233fc5c65 in service     crucible
-    425d9d2f-72ea-4f53-80b6-04e002771cdb in service     crucible
-    46a050b6-37b8-47b9-b4ae-ccb736ecd6d5 in service     crucible
-    4ea8104f-2cfc-4599-9d2c-bcfd8f0b71fd in service     clickhouse
-    7611a549-effc-41b6-b6cb-c03e0c70a31f in service     internal_ntp
-    af7ccead-2bd8-4a8a-9fbf-3b788d180139 in service     cockroach_db
-    d4428e41-ad7b-41f0-bde6-7c7c955f20c1 in service     crucible
+    0db88baf-911d-47bb-af71-3aebd7c96934 in service     nexus [underlay IP fd00:1122:3344:103::4]
+    213d7dd3-95a4-45a7-99ca-e4899991dab4 in service     crucible [underlay IP fd00:1122:3344:103::6]
+    218f36a5-9e51-4b88-ad1a-da4d85c246c4 in service     internal_dns [underlay IP fd00:1122:3344:3::1]
+    40c47ead-3be8-4724-a8ef-b66233fc5c65 in service     crucible [underlay IP fd00:1122:3344:103::7]
+    425d9d2f-72ea-4f53-80b6-04e002771cdb in service     crucible [underlay IP fd00:1122:3344:103::9]
+    46a050b6-37b8-47b9-b4ae-ccb736ecd6d5 in service     crucible [underlay IP fd00:1122:3344:103::8]
+    4ea8104f-2cfc-4599-9d2c-bcfd8f0b71fd in service     clickhouse [underlay IP fd00:1122:3344:103::5]
+    7611a549-effc-41b6-b6cb-c03e0c70a31f in service     internal_ntp [underlay IP fd00:1122:3344:103::b]
+    af7ccead-2bd8-4a8a-9fbf-3b788d180139 in service     cockroach_db [underlay IP fd00:1122:3344:103::3]
+    d4428e41-ad7b-41f0-bde6-7c7c955f20c1 in service     crucible [underlay IP fd00:1122:3344:103::a]
```

Comment:
As part of this PR I unified the way we display zones between blueprints and diffs. I tried to make the output relatively clean.

I tried to add padding to align all the fields, but then realized this would just be better as a table. I'll do that in a followup since this PR has gotten large enough already.

Reply:
Oh also, I was able to keep the sled IDs the same by using a migration script for the JSON schema.

Reply:
Tabular output is in https://github.com/oxidecomputer/omicron/pull/5270 -- it looks so much better, wow.

Reply:
(While this output has been removed in `cmd-complex-stdout`, the general principle still applies. I'd rather we just focus on #5270 to improve the output, because it is much better there.)